enterprise/frontend/src/embedding-sdk/components/private/SdkError.tsx

```diff
@@ -1,17 +1,11 @@
 import { t } from "ttag";
 
-import { useSdkSelector } from "embedding-sdk/store";
-import { getLoginStatus } from "embedding-sdk/store/selectors";
-import type { LoginStatusError } from "embedding-sdk/store/types";
-
 // TODO: Allow this component to be customizable by clients
-export const SdkError = () => {
-  const loginStatus = useSdkSelector(getLoginStatus) as LoginStatusError;
-
+export const SdkError = ({ message }: { message: string }) => {
```

Comment:
I moved the redux logic to `PublicComponentWrapper` so `SdkError` can be used in components like `StaticQuestion` and `InteractiveQuestion`. Clients can then use the same error component for all errors rather than having different error components for the public wrapper and the island components